### ISSUE TITLE
Compilation fails when absorbing boundaries are added

### ISSUE THREAD

**Issue by rgill3141** [PrincetonUniversity/tristan-mp-v2#12]
Hi,

When I add the `-absorb` flag to the configuration, like so `python configure.py -2d -mpi08 -hdf5 -extfields -absorb --user=user_alfven`, the code fails to compile and I get the following error:
```
src/algorithms/fieldsolvers/absorb_ampere.F08:41:50:

   41 |                       2.0 * lam * lam2 * ex_target
      |                                                  1
Error: Symbol 'ex_target' at (1) has no IMPLICIT type
src/algorithms/fieldsolvers/absorb_ampere.F08:48:50:

   48 |                       2.0 * lam * lam2 * ey_target
      |                                                  1
Error: Symbol 'ey_target' at (1) has no IMPLICIT type
src/algorithms/fieldsolvers/absorb_ampere.F08:55:50:

   55 |                       2.0 * lam * lam2 * ez_target
      |                                                  1
Error: Symbol 'ez_target' at (1) has no IMPLICIT type
src/algorithms/fieldsolvers/absorb_faraday.F08:41:50:

   41 |                       2.0 * lam * lam2 * bx_target
      |                                                  1
Error: Symbol 'bx_target' at (1) has no IMPLICIT type
src/algorithms/fieldsolvers/absorb_faraday.F08:48:50:

   48 |                       2.0 * lam * lam2 * by_target
      |                                                  1
Error: Symbol 'by_target' at (1) has no IMPLICIT type
src/algorithms/fieldsolvers/absorb_faraday.F08:55:50:

   55 |                       2.0 * lam * lam2 * bz_target
      |                                                  1
Error: Symbol 'bz_target' at (1) has no IMPLICIT type
```
The code compiles fine without the absorbing boundaries. This behavior is new to this version and didn't exist in the older versions. Any help will be much appreciated!

**Comment by rgill3141** [comment]
The change to the fieldsolver files was made in v2.6 on Jan 24, 2023. In the earlier versions, the target fields (e.g. ez_target, etc.) didn't appear in the fieldsolver files (Faraday and Ampere).

**Comment by haykh** [comment]
this has been resolved with the v2.7
